pubspec.yaml

```diff
@@ -67,3 +67,7 @@ environment:
 
 dev_dependencies:
   melos: ^7.0.0-dev.1
+
```

Comment:
maybe add a comment here linking to the issue as to why we need to do this?

Reply:
Since all monorepos have to do the same, is there any point in linking it?

Reply:
I'll probably actually look for it myself in a few months, I'll add it.